pandas/core/indexes/base.py

```diff
@@ -681,7 +681,47 @@ def _values(self):
         return self.values
 
     def get_values(self):
-        """ return the underlying data as an ndarray """
+        """
+        Return `Index` data as an `numpy.ndarray`.
+
+        It is a getter wrapper around `Index.values`.
+
+        Returns
+        -------
+        numpy.ndarray
+            A one-dimensional numpy array of the indexes.
+
+        See Also
+        --------
+        Index.values : The original function around which get_values wraps.
+
+        Examples
+        --------
+        Getting the `Index` values of a `DataFrame`:
+
+        >>> df = pd.DataFrame([[1, 2, 3], [4, 5, 6], [7, 8, 9]],
+        ...                    index=['a', 'b', 'c'], columns=['A', 'B', 'C'])
+        >>> df
+           A  B  C
+        a  1  2  3
+        b  4  5  6
+        c  7  8  9
+        >>> df.index.get_values()
+        array(['a', 'b', 'c'], dtype=object)
+
+        Standalone `Index` values:
+
+        >>> idx = pd.Index(['1', '2', '3'])
+        >>> idx.get_values()
+        array(['1', '2', '3'], dtype=object)
+
+        `MultiIndex` arrays also have only one dimension:
+
+        >>> midx = pd.MultiIndex.from_arrays([[1, 2, 3], ['a', 'b', 'c']],
+        ...                                  names =  ('number', 'letter'))
+        >>> midx.ndim
```

Comment:
Better to show `midx.get_values()` sow that people can see it's an ndarray of tuples.